test/jdk/java/foreign/SafeFunctionAccessTest.java

```diff
@@ -24,7 +24,7 @@
 /*
  * @test
  * @enablePreview
- * @requires ((os.arch == "amd64" | os.arch == "x86_64") & sun.arch.data.model == "64") | os.arch == "aarch64" | os.arch == "riscv64"
+ * @requires jdk.CABI != "UNSUPPORTED"
```

Comment:
what about the name `jdk.native.abi` ?

Reply:
Not so sure. The CABI is really the linker kind, not the underlying ABI ('CABI' is probably a misnomer at this point).

If CABI is bad, I think `jdk(.foreign).linker` would be better as a property name